testing/testcafe/tests/dataGrid/editing.ts

```diff
@@ -1389,3 +1389,37 @@ test('Async Validation(Batch) - Validation frame should be rendered when a neigh
     }],
   }],
 })));
+
+// T905677
+test('Rollback changes on a click on a revert button  when startEditAction is dblclick', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const dataRow = dataGrid.getDataRow(0);
+  const cell0 = dataRow.getDataCell(1);
+  const $revertButton = Selector('.dx-revert-button');
+
```

Comment:
It is better to add the **getRevertButton** to the **DataCell** class that is placed in the "testing\testcafe\model\dataGrid\data\cell.ts" file. For instance:

```ts
const CLASS = {
  ...
  revertButton: 'dx-revert-button',
};

export default class DataCell extends FocusableElement {
  ...
  getRevertButton(): Selector {
    return this.element.find(`.${CLASS.revertButton}`);
  }
}
```

After that, you will be able to get the button and use it in the test in the following manner:
```ts
const revertButton = cell0.getRevertButton();

await t
    ...
    .click(revertButton)
    .expect(revertButton.exists)
```